merge_request_approvals.go

```diff
@@ -417,3 +422,28 @@ func (s *MergeRequestApprovalsService) DeleteApprovalRule(pid interface{}, merge
 
 	return s.client.Do(req, nil)
 }
+
+// ChangeProjectMergeRequestApprovalConfiguration updates the merge request approval configuration of a project.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/merge_request_approvals.html#change-configuration
+func (s *MergeRequestApprovalsService) ChangeProjectApprovalConfiguration(pid interface{}, opt *MergeRequestApprovals, options ...RequestOptionFunc) (*MergeRequestApprovals, *Response, error) {
```

Comment:
This is not correct. `MergeRequestApprovals` is not an option struct so should only be used as return value. Reading the docs it seems this only allows you to change the number of approvals required and so it should probably get its own option struct.

Reply:
I agree, I can definitely make the required changes... Would you like to suggest a name for this new option struct, please?

Reply:
Please check the other code for examples, option struct always use the name of the method with the postfix `Options`. So in this case that should be `ChangeProjectApprovalConfigurationOptions`.